README.md

```diff
@@ -675,3 +675,42 @@ You don't need to build anything to be able to run locally, but if you need the
 ```text
 poetry build
 ```
+
+### Development - Version release
+
+You can check the current version on master with these commands:
+```text
+git checkout master
+poetry version
+```
+
+To update the version use:
```

Comment:
Perhaps replace L687-L694 with a text directing to this link: https://python-poetry.org/docs/cli/#version

Reply:
done